src/platform/testing/e2e/helpers.js

```diff
@@ -94,7 +100,9 @@ function uploadTestFile(client, fileData) {
 function createE2eTest(beginApplication) {
   return {
     'Begin application': client => {
+      client.openUrl(BASE_URL);
```

Comment:
I'm interested in seeing how this will affect test performance (since we'll be calling `openUrl` twice for every test).

Reply:
Followup: I didn't notice a significant increase in testing time for the e2e step on this PR compared to `master` builds. It's maybe a few seconds slower, but that could be within the standard deviation for builds at this time of day. Either way, it doesn't seem to be a problem. :tada: